frontend/src/scenes/userLogic.tsx

```diff
@@ -78,7 +78,11 @@ export const userLogic = kea<userLogicType<UserType, UpdateUserPayload>>({
                     }
 
                     posthog.identify(user.distinct_id)
-                    posthog.people.set({ email: user.anonymize_data ? null : user.email })
+                    posthog.people.set({
+                        email: user.anonymize_data ? null : user.email,
+                        realm: user.realm,
```

Comment:
We already set this information in the `preflightLogic` (see https://github.com/PostHog/posthog/blob/27144478f534290ee01b69d6ac5c6a53b8e05ec4/frontend/src/scenes/PreflightCheck/logic.ts#L80).